integration-tests/tests/src/utils/import-app.ts

```diff
@@ -23,7 +23,7 @@ export type ErrorResponse = { message: string; appId: never };
 export type ImportResponse = { appId: string; message: never };
 export type Response = ImportResponse | ErrorResponse;
 
-const { realmBaseUrl = "http://localhost:9090" } = environment;
+const { realmBaseUrl = "http://localhost:9090", reuseApp = true } = environment;
```

Comment:
Introduced a way to disable reuse of apps as there seem to be a race that we don't want to trigger if we can avoid it .. which we can since this is testing against baas in a docker container.